### DISCUSSION TITLE
Issues with suspend (Ctrl-Z) on Fedora Linux

### DISCUSSION THREAD

**Discussion by BladeWDR** [ghostty-org/ghostty#4220] [Feature Requests, Ideas] [answered]
So this is a bit of an odd one.

For some reason this works fine on my laptop (Arch Linux, zsh 5.9) but not on my desktop (Fedora 41, same zsh version).

If I open a program, say vim - then press Ctrl Z to suspend it and send the job into the background, it behaves oddly on my desktop. It just results in a black screen. I can still press `fg` and then enter to restore the job and get back into the program, but that's it. I don't have a shell prompt or anything.

I'm managing my zsh configuration files with dotbot, so it's the same across both systems, and the zsh version is the same.

Everything works as normal in Alacritty.

Here's the setup on my desktop:

- 3 monitors
- SwayWM / wlroots
- zsh 5.9
- Fedora 41

The software stack is the same on both apart from the distro.

**Comment by 00-kat** [comment]
Just randomly found this, `shell-integration = none` change anything?

**Reply by BladeWDR** [reply]
Tried this, same thing.

**Comment by BladeWDR** [comment, marked as answer]
I just updated to v1.1.0 and whatever the problem was before, it's fixed now.
